lib/generator.js

```diff
@@ -595,12 +596,11 @@ class Generator {
         save: false
       });
   
-      const addResult = arb[Symbol.for('resolvedAdd')];
-      if (!addResult) throw new Error('Unable to resolve the name of the added package. It was most probably not added to node_modules successfully');
-  
-      const packageName = addResult[0].name;
+      const manifest = await pacote.manifest(this.templateName);
```

Comment:
I'm only wondering if it may ever happen that manifest is undefined 🤔 if we should somehow protect from it like we did before with the workaround (yeah, I know that in my code I also had it like this)

Reply:
Added the additional line to handle the `undefined manifest` case